Packs/PhishLabs/Integrations/PhishLabsIOC/PhishLabsIOC.py

```diff
@@ -681,6 +686,26 @@ def main():
     """
     Main function
     """
+
+    params = demisto.params()
+
+    server: str = (params.get('url')[:-1]
+                   if (params.get('url') and params.get('url').endswith('/'))
+                   else params.get('url'))
+
+    client = Client(
+        base_url=str(server) + '/api/v1/',
+        user_name=params.get('credentials', {}).get('identifier'),
+        password=params.get('credentials', {}).get('password'),
+        use_ssl=not params.get('insecure', False),
+        reliability=params.get('integrationReliability')
+    )
+
+    args = {
```

Comment:
I suggest passing them directly in `command_func(client, **args)` for readability.